diesel_cli/src/infer_schema_internals/information_schema.rs

```diff
@@ -176,22 +177,27 @@ where
     use self::information_schema::tables::dsl::*;
 
     let default_schema = Conn::Backend::default_schema(connection)?;
-    let schema_name = match schema_name {
+    let db_schema_name = match schema_name {
```

Comment:
Just 
```suggestion
    let db_schema_name = schema_name.unwrap_or(&default_schema);
```
should be fine

Reply:
Following your principle of not touching too much I hadn't touched this part but I agree :)

I've also updated in the same way a few lines under.